### ISSUE TITLE
Sorting method should not be global in Android client

### ISSUE THREAD

**Issue by bremenpl** [owncloud/android#904]
Hello there,
Its just my opinion, but I think that sorting method should not be global for all folders viewed in Android client. For example, In the root owncloud folder I would like to see folders sorted in A-Z order, but when viewing pictures in the media folder, its better to see them in Newest-Oldest order. At this point if i change the sorting method for one folder, it is changed everywhere. What do you think?

**Comment by tobiasKaminsky** [comment]
Hm. 
You mean that the app remembers the setting per folder.
But then we should have also an option to enable this globally @jancborchardt 
It was implemented this way first as it is the easier way ;)

**Comment by bremenpl** [comment]
Yes, thats exacly what I mean. I think it would be a neat functionality.

**Comment by jancborchardt** [comment]
Good point.

Maybe semi-global would work even better: Remembered per folder, but initially all subfolders inherit the sorting when it was changed for a parent folder.

That way you could sort your main image folder Newest-Oldest, and all sub-image-folders would follow suit without you needing to manually switch it for every one of them.

**Comment by jesmrec** [comment]
iOS app works in this way. I agree in that, the sort method should be set up in each account.

**Comment by harshitbansal05** [comment]
@jesmrec is the issue still open?

**Comment by jesmrec** [comment]
yes, it is. The way to do that is: https://github.com/owncloud/android/issues/904#issuecomment-76710222

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
